manifest.lkml

```diff
@@ -21,18 +21,18 @@ constant: SHEETS_CAMPAIGN_TABLE {
 ## Leave the value empty (value:"") if you're not using the source, otherwise you'll get an error
 
 constant: FADS_DATASET_NAME {
-  value: "PacingDashData"
-  export: override_required
+  value: "FbAds-transfer-dataset"
+  export: override_optional
 }
 
 constant: GADS_DATASET_NAME {
-  value: "GoogleAdsPD"
-  export: override_required
+  value: "GoogleAds-Transfer-Dataset"
+  export: override_optional
 }
 
 constant: GADS_CUSTOMER_ID {
-  value: "4717384083"
-  export: override_required
+  value: "11111111111"
+  export: override_optional
```

Comment:
If the GADS CUSTOMER_ID constraint is "override_required", shouldn't it be indicated on this line?

Reply:
Hi no, the customer does not necessarily have to use a Google Ads Dataset therefore it is not necessary to have a Google Ads Customer Id assigned. So, override optional is fine.

However, reviewing the code the default value should be an empty string, something like " " only